src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -159,6 +159,46 @@ public void Dispose()
             this._cancellationTokenSourceCheckForChanges.Cancel();
         }
 
+        public async Task<long> GetUnprocessedChangeCountAsync()
+        {
+            long unprocessedChangeCount = 0L;
+
+            try
+            {
+                long getUnprocessedChangesDurationMs = 0L;
+
+                using (var connection = new SqlConnection(this._connectionString))
+                {
+                    this._logger.LogDebugWithThreadId("BEGIN GetUnprocessedChangeCountAsync");
+                    await connection.OpenAsync();
+                    this._logger.LogDebugWithThreadId("END GetUnprocessedChangeCountAsync");
```

Comment:
This should go at the end

Although given you already have logs for GetUnprocessedChangeCount below I don't think you even need to keep these ones since this method is only doing one thing.

Reply:
Need to rename `GetUnprocessedChangeCountAsync` to `OpenGetUnprocessedChangesConnection`.